### DISCUSSION TITLE
Not able to login

### DISCUSSION THREAD

**Discussion by Sathyanarayanan-Dhanuskodi** [community/community#31597] [New to GitHub] [answered]
Hi,

One of my friends just reset his laptop. His mobile also got some issues at the same time. Now he is trying to log in to GitHub. But since he lost his Microsoft authenticator app, his security code, and SSH, he was not able to enter his two-way authentication code.

Is there any other option to recover the account?

Thanks in advance. It will be a great help.

**Comment by melvincwng** [comment, marked as answer]
@Sathyanarayanan-Dhanuskodi, I would suggest your friend to contact [GitHub Support](https://support.github.com/contact/account) over here to recover his account.

Find a place where he is able to access a computer, and click on the link above. Raise a ticket, and wait a couple of days for a GitHub support engineer to get back to you.

Hope this answers your question 😊

**Reply by Sathyanarayanan-Dhanuskodi** [reply]
Thanks for the quick reply. That would help us. I sent him the link you sent. 

Thanks again

**Comment by Yagergudai** [comment]
Can I get my account named "Yagergudai"

**Reply by Yagergudai** [reply]
That is my account!
